core/src/main/java/org/mule/runtime/core/api/context/notification/NotificationUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.api.context.notification;
+
+import static org.mule.runtime.api.notification.EnrichedNotificationInfo.createInfo;
+import org.mule.runtime.api.component.Component;
+import org.mule.runtime.api.component.location.ComponentLocation;
+import org.mule.runtime.api.event.Event;
+import org.mule.runtime.api.notification.ConnectorMessageNotification;
+import org.mule.runtime.core.api.event.CoreEvent;
+
+/**
+ * Utilitary methods for working with {@link org.mule.runtime.api.notification.Notification}s
+ *
+ * @since 4.3.0
+ */
+public class NotificationUtils {
```

Comment:
This class doesn't fix the comment as it's still API. Also, you no longer need to extract this from the NotificationHelper

Reply:
Removed since it's not needed anymore.